### ISSUE TITLE
Theme re-factor - Default Search

### ISSUE THREAD

**Issue by markconroy** [localgovdrupal/localgov_base#76]
# Sample: http://localgov.dev.mark.ie/search

[] Put the templates in place in the theme
[] Move styling into localgov_base theme
[] confirm feature parity

**Comment by markconroy** [comment]
PR #85

**Comment by msayoung** [comment]
@markconroy 
I've tested locally, as your site seems to have moved on (looking good btw)

Could the Apply button display inline? 

I believe on the other theme we decided (though I don't think it was implemented) to show the button as a magnifying glass with the Apply text hidden with the `.visually-hidden` class. Happy to leave this though as it's probably best to be consistent with the other searches on the site. 

The order of the items seems to have changed from the original which had: 

1. H1
2. Search box
3. H2 - Search results
4. Showing items... 
5. Results

![image](https://user-images.githubusercontent.com/3852805/125617109-f3b33a07-9a44-4b7f-8b27-b4471409b3b0.png)
![image](https://user-images.githubusercontent.com/3852805/125617195-eaec5bad-1ef6-4fe6-a0ac-9fdb684d099c.png)

Could we have the search results view mode title as an H3 please? 

The original search results didn't have visible numbers on the search results, though this doesn't bother me particularly. I don't think we are showing them on other search results, so I guess we should be consistent. 

Not sure if this is a config issue or a theme one - some types of content (eg directory venue) don't seem to have their view mode set up correctly. eg search for collaborators 

Cheers!

**Comment by markconroy** [comment]
Hi @msayoung 

- The "submit-button" is inline (via display: flex on the parent).
- I'm going to leave the word apply on the search button, and let sub-themes decide if they want to remove it
- The order of items is set via config in the view header, so not actually part of the theming work
- The title is now set to `h3`
- The numbers are set to transparent now
- The view mode setup is a config issue. when I enabled that module, none of the content types had any view mode set up. I did a few manually just for testing
